gobblin-runtime/src/main/java/gobblin/runtime/mapreduce/MRJobLauncher.java

```diff
@@ -427,20 +429,12 @@ private void addHdfsJars(String hdfsJarFileList)
    * Prepare the job input.
    * @throws IOException
    */
-  private Path prepareJobInput(List<WorkUnit> workUnits)
+  private void prepareJobInput(List<WorkUnit> workUnits)
       throws IOException {
-    // The job input is a file named after the job ID listing all work unit file paths
-    Path jobInputFile = new Path(this.jobInputPath, this.jobContext.getJobId() + WORK_UNIT_LIST_FILE_EXTENSION);
```

Comment:
What about logging instead? The file seems mostly useless for debugging, especially because it gets deleted on exit.